client/widgets/recent-activity/activity.css.styl

```diff
@@ -7,9 +7,10 @@
   .singleRunner
     .name
       margin-left 0.2em
-  .singleRunner, .multipleRunners
-    span
-      vertical-align middle
+  span.singleRunner, span.multipleRunners
```

Comment:
While you're at it, please rename these classes with the latest CSS naming convention, i.e. `single-runner` and `multiple-runners`.